### PR TITLE
fix undo delete module

### PR DESCRIPTION
**Pull request by edgardoh**
Fixes the crash when undo a module delete. Also handles better the multi-name.

### CODE REVIEW

File: src/libs/history.c [hunk 1]
```diff
@@ -193,10 +212,29 @@ static GList *_duplicate_history(GList *hist)
...
+    {
+      dt_iop_module_t *base = get_base_module(darktable.develop, old->op_name);
+      if(base)
+      {
+        params_size = base->params_size;
+      }
+      else
+      {
+        // nothing else to do
+        fprintf(stderr, "[_duplicate_history] can't find base module for %s\n", old->op_name);
```
Comment (TurboGit) [inline src/libs/history.c]: Why the underscore just after [? Is that a dt style, I never paid attention to that.

File: src/libs/history.c [hunk 2]
```diff
@@ -279,8 +297,12 @@ static void _pop_undo(gpointer user_data, dt_undo_type_t type, dt_undo_data_t *d
       // this fixes the duplicate module when undo: hitem->multi_priority = 0;
       if (hitem->module == NULL)
       {
-        const dt_iop_module_t *base = get_base_module(darktable.develop, hitem->multi_name);
-
+        const dt_iop_module_t *base = get_base_module(darktable.develop, hitem->op_name);
+        if(base == NULL)
+        {
+          fprintf(stderr, "[_pop_undo] can't find base module for %s\n", hitem->op_name);
```
Comment (TurboGit) [inline src/libs/history.c]: same question here about the underscore

File: src/libs/history.c (hunk 1 above)
Comment (edgardoh) [reply src/libs/history.c]: Is the name of the function.
No idea if there's a standard for function naming, woudn't hurt to have one. As far as the printf, if this isn't a standard for errors that just can't happen it should be, it make it easy to see where it comes from when reported.

File: src/libs/history.c (hunk 2 above)
Comment (edgardoh) [reply src/libs/history.c]: Idem.

**Comment by TurboGit** [comment]
Perfect! Fixes a recent(?) regression as I'm sure I had tested that not so long ago.

**Pull request merged**